.github/workflows/go.yml

```diff
@@ -0,0 +1,43 @@
+name: Go
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
```

Comment:
What does the '^' symbol accomplish?
Latest golang is 1.16.
Also, do you think there's value iterating over a number of golang versions? 
I have caught a build error before when I have built on newer golang versions. I was hoping to keep a list of the last 4 minor golang versions e.g. 1.16.x, 1.15.x, etc to ensure any changes still build at least across different golang versions. The builds are in parallel so it doesn't effect the speed of the workflow.

```
jobs:
  build:
    strategy:
      matrix:
        go-version: [1.13.x, 1.14.x, 1.15.x, 1.16.x]
        os: [ubuntu-latest]
````

Reply:
In theory `^` is regex for "Start with", i.e. ^1.15 should match 1.15.x.

Your idea is good! I saw it in https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/332/files, I am just wondering if it works with the "x", can you please confirm it?

Reply:
Sure - See execution of multiple golang versions here: https://github.com/martinkennelly/network-resources-injector/actions/runs/627385214